schema/geometry_schema.json

```diff
@@ -310,14 +342,23 @@
 					"type": "number"
 				},
 				"floor_to_ceiling_height": {
-					"type": "number"
+					"type": "number",
+					"minimum": 0,
+					"exclusiveMinimum": true
 				},
 				"above_ceiling_plenum_height": {
 					"type": "number"
 				},
-				"multiplier": {
+				"floor_offset": {
```

Comment:
Why do you need to offset an entire story from the story below?  For floating stories in buildings with multipliers?  We could do this automatically based on the multiplier instead?